ksql-common/src/main/java/io/confluent/ksql/function/UdfIndex.java

```diff
@@ -316,7 +318,6 @@ boolean accepts(final Schema argument, final Map<Schema, Schema> reservedGeneric
       return Objects.equals(type, argument.type())
           && CUSTOM_SCHEMA_EQ.getOrDefault(type, (a, b) -> true).test(schema, argument)
           && Objects.equals(schema.version(), argument.version())
-          && Objects.equals(schema.parameters(), argument.parameters())
```

Comment:
Is it only the decimal type that may contain parameters in their schemas? (Trying to understand whether removing this portion of the check is safe.)

Reply:
at the moment yes - it's the only one that we use that has parameters. I think in general it makes sense to defer the parameter to check to the `CUSTOM_SCHEMA_EQ` instead of checking that they are the same.